services/skipchain/api.go

```diff
@@ -0,0 +1,130 @@
+package skipchain
+
+import (
+	"github.com/dedis/cothority/lib/network"
+	"github.com/dedis/cothority/lib/sda"
+)
+
+// Client is a structure to communicate with the Skipchain
+// service from the outside
+type Client struct {
+	*sda.Client
+}
+
+// NewClient instantiates a new client with name 'n'
+func NewClient() *Client {
+	return &Client{Client: sda.NewClient("Skipchain")}
+}
+
+// CreateRootInter creates two Skipchains: a root SkipChain with
+// maximumHeight of maxHRoot and an intermediate SkipChain with
+// maximumHeight of maxHInter. It connects both chains for later
+// reference.
+func (c *Client) CreateRootInter(elRoot, elInter *sda.EntityList, maxHRoot, maxHInter int, ver VerifierID) (root, inter *SkipBlock, err error) {
```

Comment:
In fact the only thing that is missing from the API is the `AddParentChildLink`. The rest is there, with a twist:
We cannot have a `AddData` and `AddRoster`, as we have to verify the SkipBlocks first. This is why there is a `ProposeData` and `ProposeRoster`.
The `CreateRootInter` and `CreateData` are convenience API-calls because they will probably be used in most of the cases to initialize the SkipChains. What I should do is to have those two functions use `Propose.*`.